### PR TITLE
BXMSDOC-4655 QE Review: PAM 7.5 BPMN reference

### PR DESCRIPTION
**Pull request by emmurphy1**
Details in jiras:
[BXMSDOC-4655](https://issues.jboss.org/browse/BXMSDOC-4655)
[BXMSDOC-4655](https://issues.jboss.org/browse/BXMSDOC-4689)

### CODE REVIEW

File: doc-content/enterprise-only/bpmn/bpmn-intermediate-events-ref.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-= Intermediate events 
+= Intermediate events
 Intermediate events drive the flow of a business process. Intermediate events are used to either catch or throw an event during the execution of the business process.  These events are placed between the start and end events and can also be used on the boundary of an activity, like a subprocess or a human task, as a catch event.  The boundary catch events can be configured as interrupting or non-interrupting.  An interrupting boundary catch event cancels the bound activity whereas a non-interrupting event does not.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/bpmn/bpmn-intermediate-events-ref.adoc:3]: I would change "whereas"

File: doc-content/enterprise-only/bpmn/bpmn-intermediate-events-ref.adoc [hunk 2]
```diff
@@ -79,7 +79,7 @@ A message intermediate event is an intermediate event that enables you to manage
 
 [[_timer_intermediate_event]]
 .Timer
-A timer intermediate event enables you to delay workflow execution or to trigger the workflow execution periodically. It represents a timer that can trigger one or multiple times after a specified period of time. When the timer intermediate event is triggered, the timer condition (the defined time) is checked and the outgoing flow is taken. 
+A timer intermediate event enables you to delay workflow execution or to trigger the workflow execution periodically. It represents a timer that can trigger one or multiple times after a specified period of time. When the timer intermediate event is triggered, the timer condition (the defined time) is checked and the outgoing flow is taken.
```
Comment (michelehaglund) [inline doc-content/enterprise-only/bpmn/bpmn-intermediate-events-ref.adoc]: Consider rewording "the timer condition (the defined time) is checked and the outgoing flow is taken." 

to

the timer condition, which is the defined time,, is checked and the outgoing flow is used.

There are similar refs throughout. May be fine as is, but "taken" seems a bit off.

**Review by michelehaglund — APPROVED** [review]
Minor suggestions added. Otherwise looks good. No show-stoppers.

File: doc-content/enterprise-only/bpmn/bpmn-intermediate-events-ref.adoc (hunk 2 above)
Comment (emmurphy1) [reply doc-content/enterprise-only/bpmn/bpmn-intermediate-events-ref.adoc]: I will address this in the next update because it requires QE review.

**Pull request merged**